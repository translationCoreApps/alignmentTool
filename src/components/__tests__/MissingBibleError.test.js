import React from 'react';
import renderer from 'react-test-renderer';
import MissingBibleError from '../MissingBibleError';

describe('MissingBibleError', () => {
  it('renders', () => {
    const wrapper = renderer.create(
      <MissingBibleError translate={k=>k}/>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
