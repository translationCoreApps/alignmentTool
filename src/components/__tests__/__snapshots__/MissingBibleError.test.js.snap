// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MissingBibleError renders 1`] = `
<div
  id="AlignmentGrid"
  style={
    Object {
      "alignContent": "flex-start",
      "backgroundColor": "#ffffff",
      "display": "flex",
      "flexGrow": 2,
      "flexWrap": "wrap",
      "overflowY": "auto",
      "padding": "0px 10px 10px",
    }
  }
>
  <div
    style={
      Object {
        "flexGrow": 1,
      }
    }
  >
    <div
      style={
        Object {
          "backgroundColor": "#ccc",
          "display": "inline-block",
          "padding": "20px",
        }
      }
    >
      pane.missing_verse_warning
    </div>
  </div>
</div>
`;
